style.css

```diff
@@ -470,6 +475,84 @@ button:active {
   margin-left: 10px;
 }
 
+.nav-menu1 {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  gap: 20px;
+  flex-direction: column;
+  background-color: #6070ff;
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  text-align: center;
+  transition: 0.3s;
+  opacity: 0.95;
+}
+
+/* .bar1{
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease;
+  transition: all 0.3s ease;
+  background-color: white;
+  color: red;
+} */
+.nav-link {
+  margin-top: 30px;
+  height: 24px;
+  left: 100px;
+  row-gap: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: white;
+  text-decoration: none;
+}
+
+#about {
+  margin-left: -30px;
+}
+
+#nav-menu {
+  margin-left: -300px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- Kindly, the mobile menu list has been given a negative margin which leads to a lousy user experience please fix it to stick to the Figma design and have a better UI 👍
- kindly,the mobile menu close button is missed, please fix it to have the functionality👌
![2023-04-18_13-41](https://user-images.githubusercontent.com/52010789/232753283-f9c54f6e-52a7-4136-b9c8-bc0b6e4cfc86.png)